Revision Tests/Bohdanov_revision.md

```diff
@@ -0,0 +1,31 @@
+## Please translate into English
+
+**_1. Если бы я был умнее (по жизни) то я бы сделал вчера бэкап всей базы и только потом применял свой ПР к коду на проде. Жаль что я не умный как мой папа._**
+
+> If I had been smarter(in life), I would have backed up the whole base and only after it applied my pull request to the code on the prod. I wish I was smart as my dad.
+
+**_2. Если бы все студенты читали теорию и только потом писали перевод анализируя предложения, то прогресс бы был очевиден. Жаль что все хотят выучить английский брутфорсом, минуя ранние этапы обучения и анализа и отработки навыков до автоматизма. Я бы хотел чтобы вы не торопились._**
+
+> If all students read a theory and only then wrote a translation by analyzing the sentences, the progress would be obvious. If only they wouldn't tried to learn english by brutforcing, bypassing the early stages of learning and practicing skills to automation. I want you to not hurry up.
+
+**_3. Если бы сегодня мне не нужно было садиться за руль, то вчера бы я знатно набрался с друзьями в ближайшем баре и орал бы в микрофон свою любимую песню. Жаль что автопилот не является повсеместной функцией во всех автомобилях, это могло бы значительно разнообразить досуг водителей._**
+
+> If I didn't have to drive today, yesterday I would have got (gotten?) drunk with my friends in the nearest bar and screamed in the mic my favorite song.I wish an autopilot was generic function in all cars, it could significantly diversify drivers free time. 
```

Comment:
there is no form "gotten"
a generic function
all the cars
drivers' free time

Reply:
https://www.grammarly.com/blog/got-gotten/